src/NuGet.Core/NuGet.Protocol/VisualStudio/ExtensibleSourceRepositoryProvider.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#if IS_DESKTOP
```

Comment:
It was merged in from NuGet.Protocol.VisualStudio which was a net45 project. NuGet.Protocol is a net45/netstandard1.3 project. 

The desktop framework requirement comes from using MEF here, this class is only used in the VSIX.